libtenzir/src/detail/installdirs_relocatable.cpp.in

```diff
@@ -26,6 +28,9 @@ std::filesystem::path install_libdir() {
 /// directories for multi-config generators.
 std::pair<std::filesystem::path, std::filesystem::path>
 install_prefix_and_suffix() {
+  if (auto runtime_prefix = detail::getenv("TENZIR_RUNTIME_PREFIX")) {
+    return {*runtime_prefix, {}};
+  }
```

Comment:
Does this need to be added to `application.cpp` so that it's also available via `tenzir-node --runtime-prefix=...`?

Reply:
Makes sense.